src/builtins/BuiltinTemporal.cpp

```diff
@@ -16,7 +16,7 @@
  *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301
  *  USA
  */
-
+#include <array>
```

Comment:
Is this needed?

Reply:
We need to include libraries only when there is no other option.
Of course, standard libraries give us many features we can easily use, but sometime it may slowdown the overall performance or increase the binary size. In most cases, we need a few simple functions of library but usually these functions consists of a lot of features for safety purpose, which looks quite excessive for our implementation.

It seems that you assign `std::array` just for arguments, and IMO it can be fully replaced with a structure or argument list.

Reply:
Is it necessary to use std::array here?
IMO it seems that std::array is used for simple argument array, so another simple structure or stack-allocated array could replace it.